<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="m_playButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="myplayer.qrc">
     <normaloff>:/images/play.png</normaloff>:/images/play.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="m_openButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="myplayer.qrc">
     <normaloff>:/images/folder.png</normaloff>:/images/folder.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="m_stopButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="myplayer.qrc">
     <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
   </property>
  </widget>
  <widget class="QSlider" name="m_volumeSlider">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>170</y>
     <width>29</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="m_deleteButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="myplayer.qrc">
     <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
   </property>
  </widget>
  <widget class="QListWidget" name="m_playlist">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_loopButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="myplayer.qrc">
     <normaloff>:/images/loop.png</normaloff>:/images/loop.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="m_randomButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="myplayer.qrc">
     <normaloff>:/images/random.png</normaloff>:/images/random.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="m_endButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>end</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_prevButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="myplayer.qrc">
     <normaloff>:/images/prev.png</normaloff>:/images/prev.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>23</width>
     <height>23</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="m_nextButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="myplayer.qrc">
     <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>23</width>
     <height>23</height>
    </size>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myplayer.qrc"/>
 </resources>
 <connections/>
</ui>
